arroyo-rpc/src/lib.rs

```diff
@@ -125,3 +127,16 @@ pub fn primitive_to_sql(primitive_type: PrimitiveType) -> &'static str {
         PrimitiveType::Json => "JSONB",
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct RateLimit {
+    pub messages_per_second: u32,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct OperatorConfig {
```

Comment:
It's in arroyo-rpc instead of arroyo-types because it needs `serde_json::Value` which arroyo-types doesn't currently have, and ideally we don't add additional crate dependencies to arroyo-types.

Generally I think arroyo-rpc makes sense for values that are used in communication between the various parts of our system.